docs/tutorial/truncating.md

```diff
@@ -0,0 +1,281 @@
+# Truncating
+
+Table lookups have a strict constraint on the number of bits they support. This can be limiting, especially if you don't need exact precision. As well as this, using larger bit-widths leads to slower table lookups.
+
+To overcome these issues, truncated table lookups are introduced. This operation provides a way to zero the least significant bits of a large integer and then apply the table lookup on the resulting (smaller) value.
```

Comment:
not sure here ? are they only zeroised ? or are they discarded/cropped ?
the sentence just after say the precision is reduced.

Reply:
Yeap, they are only zeroed.